src/input/component.rs

```diff
@@ -551,13 +551,15 @@ where
                 }
                 self.remove_required_msg(&id, txt.is_empty());
                 self.unique_msg.remove(&id);
+                self.decide_unique_all(ctx);
```

Comment:
Thanks for the update! The call to decide_unique_all(ctx) inside the Message::InputText branch looks unrelated to this issue, so I’d prefer to keep the text-input behavior unchanged in this PR. Could you drop that addition? Everything else around the domain-name handling looks great, and I confirmed that the “already exists” message now appears immediately when a duplicate domain is entered.